### DISCUSSION TITLE
understanding-samesite-cookies/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#379] [Announcements]
# understanding-samesite-cookies/

In this post I discuss SameSite cookies, what they are, why they're useful, and the limitations when you use them.

https://andrewlock.net/understanding-samesite-cookies/

**Comment by rosdi** [comment]
Wow this explains a lot! This will greatly help me troubleshooting any issues with authentication or cookies.

**Comment by drdamour** [comment]
wish this was around 5 years ago wheni was fighting that safari ?11? or was it 10? stuff with SameSite :(

**Comment by kzimny** [comment]
Thank you! It's very good explained. It helped me a lot.
